traitsui/testing/tester/ui_tester.py

```diff
@@ -38,6 +38,12 @@ class UITester:
         Time delay (in ms) in which actions by the tester are performed. Note
         it is propagated through to created child wrappers. The delay allows
         visual confirmation test code is working as desired. Defaults to 0.
+    process_events : bool, optional
```

Comment:
A more unique name would be good... `do_process`, `auto_process`, ...?

Reply:
Between those I like `auto_process` more.  Thoughts on `auto_process_events`? I think it is okay if it is similar to the `process_events` callable as they are related, but they shouldn't be identical.  I feel like events carries some information though, but if you disagree I'm happy with either.

Reply:
`auto_process_events` sounds good to me.
The default value is the most common use case so I'm happy for the name to be longer so that it is explicit and clear what it does.